bricks/net/tcp/impl/posix.h

```diff
@@ -199,24 +249,27 @@ class SocketHandle : private SocketSystemInitializer {
  private:
   SocketHandle() = delete;
   void operator=(const SocketHandle&) = delete;
-  void operator=(SocketHandle&&) = delete;
+  void operator=(SocketHandle&& rhs) {
+    socket_ = rhs.socket_;
+    rhs.socket_ = static_cast<SOCKET>(-1);
+  }
 
-#ifndef CURRENT_WINDOWS
+#if 1  // ndef CURRENT_WINDOWS
```

Comment:
Looks like it works on Windows well? Do we still need an else part?

Reply:
Yes, I've checked this on Windows. Let me check again and remove the `#else` one =)

Reply:
Thanks for pointing this our, Max: This particular piece was now OK to remove, but the test that opens two sockets on one port -- another place I've tried again with Windows -- still fails, so I've changed that too.

Waiting for the last commit to (hopefully) turn green on Travis and pushing the update.

Reply:
:+1: f9d4dcaa54b179c9995f2a1a7c38cd74496146af

Thanks for pointing this out, Max: This particular piece was now OK to remove, but the test that opens two sockets on one port -- another place I've tried again with Windows -- still fails, so I've changed that too.